pipelines/release-to-github-signed/README.md

```diff
@@ -0,0 +1,18 @@
+# Release to GitHub Signed Pipeline
+
+Tekton release pipeline to release binaries extracted from the image built with RHTAP, signed a checksum file, and release the binaries and signed file to GitHub.
+
+## Parameters
+
+| Name                            | Description                                                                                              | Optional  | Default value                                   |
+|---------------------------------|----------------------------------------------------------------------------------------------------------|-----------|-------------------------------------------------|
+| release                         | The namespaced name (namespace/name) of the Release custom resource initiating this pipeline execution   | No        | -                                               |
+| releaseplan                     | The namespaced name (namespace/name) of the releasePlan                                                  | No        | -                                               |
+| releaseplanadmission            | The namespaced name (namespace/name) of the releasePlanAdmission                                         | No        | -                                               |
+| snapshot                        | The namespaced name (namespace/name) of the snapshot                                                     | No        | -                                               |
+| enterpriseContractPolicy        | JSON representation of the EnterpriseContractPolicy                                                      | No        | -                                               |
+| enterpriseContractPublicKey     | Public key to use for validation by the enterprise contract                                              | Yes       | k8s://openshift-pipelines/public-key            |
+| verify_ec_task_git_url          | The git repo url of the verify-enterprise-contract task                                                  | No        | -                                               |
```

Comment:
These three ec parameters were replaced with just the bundle. You probably copied a pipeline before the change, but can you switch it to be like the others now?

Reply:
Replaced! Thank you!